src/hotspot/share/runtime/continuationFreezeThaw.cpp

```diff
@@ -2154,14 +2150,12 @@ NOINLINE void ThawBase::recurse_thaw_interpreted_frame(const frame& hf, frame& c
 
   assert((stack_frame_bottom >= stack_frame_top + fsize) &&
          (stack_frame_bottom <= stack_frame_top + fsize + 1), ""); // internal alignment on aarch64
```

Comment:
Your're right @pchilano, it should. But since the line just above is:
`const int fsize = heap_frame_bottom - heap_frame_top;`
I thought that there was no use in keeping the `assert` at all, so I removed it.

Reply:
Yes, but it's comparing it with the created stack frame's size so I think it doesn't hurt to keep it. We have an equivalent assert in recurse_freeze_interpreted_frame().

Reply:
Put back the assert.